contrib/opencensus-correlation/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '0.2.dev0'
+__version__ = '0.1.0'
```

Comment:
Same here.

Reply:
There is no change, probably no need to release this package.

Reply:
Just noticed this one is special, the version should be 0.3.0 although it hasn't been changed at all.

Reply:
Changed to 0.3.0.